src/controls/Button.jsx

```diff
@@ -35,60 +36,53 @@ const buttonStyles = {
  * @param {Object} [props.style] Desired JSX-formatted styling object; overrides all other style attributes
  * @returns {import("react").ReactComponentElement}
  */
-export default function Button(props) {
-  let {
-    text,
-    onClick,
-    width,
-    type,
-    size,
-    height,
-    hoverStart,
-    backgroundColor,
-    activeBackgroundColor,
-    style,
-  } = props;
+const Button = (props) => {
+  let { text, onClick, width, type, size, height, style } = props;
 
   // Assign fallback values for styling
   if (!type || !(type in Object.keys(buttonStyles))) {
     type = "default";
   }
 
   return (
-    <HoverBox
-      hoverBackgroundColor={
-        activeBackgroundColor || buttonStyles[type].hoverBackgroundColor
-      }
-      hoverStart={hoverStart || "left"}
+    <AntButton
       style={{
-        alignItems: "center",
         display: "flex",
-        backgroundColor:
-          backgroundColor || buttonStyles[type].standardBackgroundColor,
-        color: "white",
+        borderRadius: "0",
+        backgroundColor: colors.TEAL_ACCENT,
```

Comment:
```suggestion
        backgroundColor: buttonStyles[type].standardBackgroundColor,
```
This is to return the ability to set buttons of different types, like "disabled" or "secondary"